### PR TITLE
add link to new docs

### PR DESCRIPTION
**Pull request by SteffenHeu**
Added a constructor parameter to the SimpleParameterSet to open the new documentation link (if available)

### CODE REVIEW

File: src/main/java/io/github/mzmine/parameters/impl/SimpleParameterSet.java [hunk 1]
```diff
@@ -204,7 +210,7 @@ public ExitCode showSetupDialog(boolean valueCheckRequired) {
     if ((parameters == null) || (parameters.length == 0)) {
       return ExitCode.OK;
     }
-    ParameterSetupDialog dialog = new ParameterSetupDialog(valueCheckRequired, this);
+    ParameterSetupDialog dialog = new ParameterSetupDialog(valueCheckRequired, this, null);
```
Comment (robinschmid) [inline src/main/java/io/github/mzmine/parameters/impl/SimpleParameterSet.java:213]: This should actually copy the url as well - instead of null

**Comment by robinschmid** [comment]
Is it safe to use the parameterset here?
Or is there any chance the url gets lost.

- [ ] clone parameterset is safe
- [ ] write and read of parameters is safe

File: src/main/java/io/github/mzmine/parameters/impl/SimpleParameterSet.java (hunk 1 above)
Comment (SteffenHeu) [reply src/main/java/io/github/mzmine/parameters/impl/SimpleParameterSet.java:213]: What do you mean? That's the header message

**Comment by SteffenHeu** [comment]
No clue what you mean, but i remembered the cloning

**Comment by robinschmid** [comment]
Good like this. We are safe using it in the parameter classes as its only used in the dialogs.

**Pull request merged**